azerothcore/azerothcore-wotlk
# fix(DB/smart_scripts): Open door on Sneed's death

## CHANGES PROPOSED:
-  Add SmartScript to open Heavy Door on Sneed's death in Deadmines
- [See this video for proof](https://youtu.be/rCmsbgZIKLA?t=77) that the door is supposed to open automatically.

## ISSUES ADDRESSED:
- Door doesn't open automatically when Sneed dies in Deadmines

## TESTS PERFORMED:
- Apply SQL
- Run Deadmines
- Kill Sneed

## HOW TO TEST THE CHANGES:
- Apply SQL, restart server, create character
- `.tele deadmines`, run instance
- Kill Sneed (second boss), watch the door - it should open automatically like the first boss opens the first door automatically.


## Target branch(es):
